### ISSUE TITLE
[Question]: 使用chatglm的多卡全参数微调是出现问题，每次运行到百分之三就停止

### ISSUE THREAD

**Issue by dingjq1216** [PaddlePaddle/PaddleNLP#6754]
### 请提出你的问题

ValueError :( InvalidArgument ) Input ( Logits ) and Input ( Label ) should in same shape In dimensions except axis .
 Hint : Expected logits _ dime [ i ]== labels _ dime [ i ], but received logita _ dime [ i ]:56 f = labels _ dins [ i ]:252.]( at ../ paddle / phi / inferneta / binary . cc :909) LAUNCH INPO 2023-08-1620:46:23,137 Exit code -15
这是当时的报错，记得两次都是跑到100/3000是时候停止
GPU是四卡V100
python -m paddle.distributed.launch --gpus "0,1,2,3" finetune_generation.py \
--model_name_or_path THUDM/chatglm-6b \
--task_name_or_path ../FieldOfMachinery/ \
--max_steps 3000 \
--learning_rate 3e-5 \
--warmup_steps 20 \
--eval_steps 100 \
--logging_steps 1 \
--save_steps 1000 \
--save_total_limit 1 \
--output_dir ./checkpoints/chatglm-6b-finetune \
--src_length 64 \
--tgt_length 64 \
--per_device_eval_batch_size 4 \
--per_device_train_batch_size 4 \
--gradient_accumulation_steps 32 \
--fp16 \
--fp16_opt_level O2 \
--recompute True \
--do_train \
--do_eval \
--load_best_model_at_end True \
--tensor_parallel_degree 4 \
--do_generation True

运行单卡lora微调和单卡prefix微调时候没有问题，这个应该如何解决

**Comment by w5688414** [comment]
推荐使用A100的显卡，另外检查一下您的内存和显存是否足够。
